docs/content/how-to/azure/create-azure-cluster-with-options.md

```diff
@@ -111,7 +111,7 @@ To enable the ephemeral OS disk option on the Azure VMs in your HostedCluster, s
     You may need to adjust the disk storage account type; to adjust the disk storage account type, 
     use the `disk-storage-account-type` flag as shown in the example below.
     
-    You may need to adjust the disk size depending on the instance type used; to adjust the disk size, use the 
+    The OS disk size defaults to 120GB. You may need to adjust the disk size depending on the instance type used; to adjust the disk size, use the 
```

Comment:
This is not true. It defaults to 30Gb. https://github.com/openshift/hypershift/blob/0dd3be4c8747989c5e5b0e540649fba0503cf4d2/api/hypershift/v1beta1/nodepool_types.go#L873

Reply:
The required disk size is based on the instance type you use. Maybe you could rewrite this to say `The default size disk size of the instance type, Standard_DS4_v2, is 120GB`. You should update the examples to show the updated disk size flag setting this.

Reply:
Reverted this phrase to its original state.